src/docs/stan-reference/language.tex

```diff
@@ -1807,6 +1809,123 @@ \subsection{Operator Precedence and Associativity}
 is interpreted as \mbox{\code{-(n \textasciicircum\ 3)}}.
 
 
+\section{Conditional Operator}\label{conditional-operator.section}
+
+\subsection{Conditional Operator Syntax}
+
+The ternary conditional operator is unique in that it takes three
+arguments and uses a mixed syntax.  If \code{a} is an expression of
+type \code{int} and \code{b} and \code{c} are expressions that can be
```

Comment:
I don't think this is true. I think `\code{a}` can be of type `real`, but evaluate to non-zero and that'll be treated as `true`, right? (that's what you say below)